tests/report.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+import easypost
+from constants import API_KEY as api_key
+
+easypost.api_key = api_key
+
+class ReportTests(unittest.TestCase):
```

Comment:
It'd be great if we could get some tests that test create for the other two types and all/retrieve for all three